src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -29,8 +29,15 @@ make those filters work as expected.
 
 ## Extended data types and schema scopes
 
-There are various kinds of extended data. Listings support public data,
-private data, and public metadata. All these are editable in Console by
+There are various kinds of extended data. Search schema support differs depending on whether you
+use Marketplace API or Integration API, and on whether the `schema-for` value is `listing` or `user`. The following table summarizes the supported search schema scopes.
+
+|          | Marketplace API                | Integration API                  |
+| -------- | ------------------------------ | -------------------------------- |
+| listing  | public, meta, private, keyword | public, meta, keyword (private, protected coming in the future) |
+| user     | N/A                            | public, meta, private, protected |
+
+For extended data exposed via Marketplace API, they are editable in Console by
```

Comment:
This isn't exactly true. All extended data is editable in Console, regardless of whether it's exposed in the Marketplace API or not.

Reply:
Fixing it. `but only public data and metadata can be seen by other users.` this part may also need some advice.